docs/_includes/donut-chart.html

```diff
@@ -0,0 +1,98 @@
+<div id="donut-chart">
+    {% include section-header.html name="donut-chart" version=page.module_metadata.donut-chart.ds-component.version %}
+
+    <p>
+        The donut chart is a data visualization module designed to display comparative data in an easily digestible and
+        visually appealing format. Our donut chart is composed of four sections: the title, metric, graph, and
+        legend, each serving a distinct purpose to enhance the user's understanding of the data presented.
+    </p>
+    
+    <p>
+        In practice, the `.donut-chart__graph` is where we will load in our Highcharts donut chart.
+    </p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="donut-chart">
+                <div class="donut-chart__grid">
+                    <div class="donut-chart__title">
```

Comment:
I think this should be a heading; difficult to say which heading level, as it matters how deeply nested it is. In the context of the Skin docs, this would be an H2.